src/cli/init.rs

```diff
@@ -70,32 +101,56 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     // Fail silently if it already exists or cannot be created.
     fs::create_dir_all(&dir).ok();
 
-    // Write pixi.toml
-    let name = dir
-        .file_name()
-        .ok_or_else(|| {
-            miette::miette!(
-                "Cannot get file or directory name from the path: {}",
-                dir.to_string_lossy()
-            )
-        })?
-        .to_string_lossy();
-    let version = "0.1.0";
-    let author = get_default_author();
-    let channels = if let Some(channels) = args.channels {
-        channels
+    let (name, channels, conda_deps, pypi_deps) = if let Some(env_file) = args.env_file.clone() {
```

Comment:
I think this should be moved to a seperate function.

Reply:
Gave it a try, checkout how it looks now. Moving out more creates an even greater parameter shit, for that i would like to improve it completely with implementing stuff for ProjectManifest. But that makes this more work.